hawc/apps/assessment/forms.py

```diff
@@ -139,6 +139,7 @@ class Meta:
             "enable_risk_of_bias",
             "enable_bmd",
             "enable_summary_text",
+            "epi_version",
```

Comment:
in addition to including here, you probably need to exclude from the class AssessmentForm above